ntbs-service/DataAccess/DataQualityRepository.cs

```diff
@@ -102,6 +104,39 @@ public async Task<IList<Notification>> GetNotificationsEligibleForDotVotAlerts()
                 .ToListAsync();
         }
 
+        public async Task<IList<NotificationAndDuplicateIds>> GetNotificationIdsEligibleForPotentialDuplicateAlerts()
+        {
+            return await GetBaseNotificationQueryableForAlerts()
+                .SelectMany(a => _context.Notification, 
```

Comment:
```suggestion
                .SelectMany(_ => _context.Notification, 
```
Makes it more explicit that we're not using that value :)